packages/shared/src/lib/core/nfts/stores/active-profile-collections-per-account.store.ts

```diff
@@ -0,0 +1,42 @@
+import { Writable, get, writable } from 'svelte/store'
+import { persistedCollections } from '.'
+import { Collections } from '../types'
+import { Nft } from '../interfaces'
+import { NftStandard } from '../enums'
+
+export const collectionsSearchTerm = writable('')
+
+export const activeProfileCollectionsPerAccount: Writable<{
```

Comment:
we need to reset this store on logout, else every logout -> login will increase the NFTs in the collection